packages/synapse-react-client/src/components/EntityTreeTable/components/EntityTreeTableView.test.tsx

```diff
@@ -0,0 +1,260 @@
+import React from 'react'
+import { render, screen } from '@testing-library/react'
+import { flexRender } from '@tanstack/react-table'
+import { EntityTreeTableView } from './EntityTreeTableView'
+import { EntityTreeTableContext } from './EntityTreeTableContext'
+
+// Mock dependencies
+vi.mock('@tanstack/react-table', () => ({
+  flexRender: vi.fn(),
+}))
+
+vi.mock('./AutoLoadMore', () => ({
+  AutoLoadMore: vi.fn(({ onLoadMore, depth, isLoading }) => (
+    <div data-testid="auto-load-more">
+      <button onClick={onLoadMore}>Load More</button>
+      <span>Depth: {depth}</span>
+      <span>Loading: {isLoading.toString()}</span>
+    </div>
+  )),
+}))
+
+vi.mock('../../TanStackTable/StyledTanStackTable', () => ({
+  default: vi.fn(({ table, slots, styledTableContainerProps }) => {
```

Comment:
I think providing this full mock implementation is unnecessary and leads you to testing mock behavior. 

I think one way to simplify this is to split the tests for the row and the full view (I elaborated [here](https://github.com/Sage-Bionetworks/synapse-web-monorepo/pull/2236/files#r2395065994))